docs/usage/controls-shortcuts.md

```diff
@@ -0,0 +1,38 @@
+## UI
+
+| Shortcut                       | Description                                                                                |
```

Comment:
The description column switches back and forth between the imperative and realis moods. IMO it should all be in the imperative mood, but either way it should be consistent.

Reply:
tried to resolve it, lmk if I did.